cpp/plugins/cucim.kit.cuslide/src/cuslide/jpeg/libjpeg_turbo.cpp

```diff
@@ -1,18 +1,6 @@
 /*
- * Apache License, Version 2.0
- * Copyright 2020 NVIDIA Corporation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * SPDX-FileCopyrightText: Copyright (c) 2020-2025, NVIDIA CORPORATION
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
See https://github.com/rapidsai/cucim/pull/960/files#diff-1897ab326b175400fe6941b6a8e7c1b13d95078ce800cb4814f8bda199821532R7-R14 below (line 7-15)
```
/**
 * Code below is derived from the libjpeg-turbo's code which is under three compatible
 * BSD-style open source licenses
 * - The IJG (Independent JPEG Group) License
 * - The Modified (3-clause) BSD License
 * - The zlib License
 * Please see LICENSE-3rdparty.md for the detail.
 *  - https://github.com/libjpeg-turbo/libjpeg-turbo/blob/00607ec260efa4cfe10f9b36d6e3d3590ae92d79/tjexample.c
 *  - https://github.com/libjpeg-turbo/libjpeg-turbo/blob/2.0.6/turbojpeg.c#L1241
 */
```

I would like to suggest the following:
```suggestion
 * SPDX-FileCopyrightText: Copyright (C)2009-2020 D. R. Commander.
 * SPDX-FileCopyrightText: Copyright (c) 2020-2025, NVIDIA CORPORATION
 * SPDX-License-Identifier: BSD-3-Clause-Modification
 * SPDX-License-Identifier: Apache-2.0
```

Reply:
I also included the Zlib and IJG licenses for this one.